backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/backup/StopVmBackupCommand.java

```diff
@@ -70,7 +70,7 @@ protected void executeCommand() {
             vmBackup.setDisks(vmBackupDao.getDisksByBackupId(vmBackup.getId()));
 
             VmBackupPhase phase = vmBackup.getPhase();
-            if (phase.isBackupFinalizing() || phase.isBackupFinished()) {
+            if (phase.isBackupFinalizing() || phase.isBackupFinished() || vmBackup.isStopped()) {
```

Comment:
how about generating a different error message in case `vmBackup.isStopped()==true` (since the phase of the backup may not change yet)